test/reference/data.py

```diff
@@ -54,5 +54,11 @@ def new():
     d.basic_choice = BasicChoice(d.choice_selector)
     d.basic_choice.has_a = 5
 
+    # The next test checks that float types are correctly stored in packed and
+    # nonpacked arrays
+    d.float_member = 23.5
+    for i in range(100):
+        d.float_array.append(float(i))
+
```

Comment:
Could we have a separate folder (similar to `test/reference`), which only tests the float type packing?

Reply:
Yes, we had this wish already a few times, and I fully agree that we should do this. However, I am scared of Bazel, that's why I would like to shift it a bit to a point when we have more time.
I will start a draft PR with low priority to work on that over time.